webapp/src/Service/ExternalContestSourceService.php

```diff
@@ -287,7 +289,13 @@ protected function importFromCcsApi(array $eventsToSkip, ?callable $progressRepo
                         $event = $this->dj->jsonDecode($line);
                         $this->importEvent($event, $eventsToSkip);
 
-                        $this->setLastEvent($event['id']);
+                        $newFormat = !isset($event['op']);
```

Comment:
Maybe use a slightly more explicit variable name (since this is used far away in the code), for example `$event_format_202207`?
In the future we might even want to read/detect the exact version and store it in a variable with the same constants https://github.com/DOMjudge/domjudge/pull/1668/files#diff-efa9e2dd8b5fccf408219fb062d6e943b7c49fa715163a4a600fbb23c941bae1R51 ...(?)